src/Tribe/Template.php

```diff
@@ -507,7 +507,7 @@ public function template( $name, $context = array(), $echo = true ) {
 		static $template_names = [];
 
 		// Key we'll use for in-memory caching of expensive operations.
-		$cache_name_key = $name;
+		$cache_name_key = is_array( $name ) ? end( $name ) : $name;
```

Comment:
```suggestion
		$cache_name_key = is_array( $name ) ? implode( '/', $name ) : $name;
```

Reply:
It's called a few times and the one that's different is the last item, that's why I'm grabbing that one. We can go either way.